x-pack/plugins/security_solution/public/management/pages/event_filters/view/components/flyout/index.tsx

```diff
@@ -204,7 +214,7 @@ export const EventFiltersFlyout: React.FC<EventFiltersFlyoutProps> = memo(
         </EuiFlyoutHeader>
 
         <EuiFlyoutBody>
-          <EventFiltersForm allowSelectOs={!data} />
+          <EventFiltersForm allowSelectOs={!data} policies={policiesRequest?.data?.items ?? []} />
```

Comment:
we discussed this with Paul and we agreed we don't want to show the flyout before the form is fully completed to show up.

This can cause situations where the user can submit the form in a "loading" state, before the policies finished loading

Reply:
in the host isolation exceptions case, the flyout is only displayed after the policies request was completed.